gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -219,7 +219,7 @@ gpii.settingsHandlers.changesToSettings = function (element) {
     return element.type === "DELETE" ? undefined : element.path ? {
         path: element.path,
         value: element.value
-    } : element.value;
+    } : fluid.isPrimitive(element) ? element : element.value;
```

Comment:
This is a bit funny, what case does this end up being tripped in? In general this kind of "flexible schema" leads to issues and we should try to retain the invariant that the payload is an Object with the value in "value" if possible